### PR TITLE
Simplified some sentences.

### PR DESCRIPTION
**Pull request by aditimaurya**
I've simplified some complex sentences, which seems a little bit tough to read. I replaced "in the sequential manner" with "sequentially" which makes the whole sentence accurate and straightforward. Apart from this, I have also corrected a sentence which seems grammatically incorrect to me. I hope it will help to improve the documentation.

### CODE REVIEW

**Review by thealphadollar — CHANGES_REQUESTED** [review]
Hey Aditi, thanks for the pull request. Do you think it would be possible for you to add a few lines of description in the PR? Just 1-2 lines of what this change is should be good.

I'm attaching a good resource for understanding how PR descriptions are written. This is an overkill for this PR, and just skimming through the article is good enough. You'll get a hang of it as you contribute more, so please don't worry about it much.

https://www.pullrequest.com/blog/writing-a-great-pull-request-description/

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -8,15 +8,15 @@ Follow the golder rule:
...
+The first thing to do always is to install the extension and note how it works. One of the best ways to do this is to launch the application and go through the files that it launches sequentially.
 
 Also take a note of the coding style and dependencies that the project follows.
 
 A helpful resource for this project is https://developer.chrome.com/extensions/faq
 
 ## Choosing issues
 
-It is recommended to go through the issues tab once you have ran the program once and look for easy issues.
+It is recommended to go through the issue tabs, then run the program once and look for the easy issues.
```
Comment (thealphadollar) [inline CONTRIBUTING.md:19]: Hey Aditi, I think this modification changes the order in which a potential contributor is to contribute. Can you please have a look?

File: README.md [hunk 2]
```diff
@@ -1,7 +1,7 @@
 IITKGP ERP AUTO LOGIN
 =====================
 
-A chrome extension that you have to setup just one time, to automatically login to your IITKGP ERP account every time. Just [install the chrome extension](https://chrome.google.com/webstore/detail/iitkgp-erp-auto-login/gdickphilmdekcbmpjmbnbikchaecbdk), and click on the extension logo (IIT Kharagpur's logo) to setup your login credentials.
+You have to set up a chrome extension once, in order to login your IIT KGP ERP account every time. Just [install the chrome extension](https://chrome.google.com/webstore/detail/iitkgp-erp-auto-login/gdickphilmdekcbmpjmbnbikchaecbdk), and click on the extension logo (IIT Kharagpur's logo) to setup your login credentials.
```
Comment (thealphadollar) [inline README.md:4]: I feel here also the sense of the sentence is altered - the modification sounds like we are talking about "some extension" while the section aims at describing what IITKGP ERP AUTO LOGIN is. Can you please spare some time to look through the modification and see, if any, changes are required?

**Review by thealphadollar — COMMENTED** [review]
Yoohoo, the description looks solid. Really glad to have a contribution from you, @aditimaurya  .

I've left a few comments, please have a look at them as you are comfortable.

File: CONTRIBUTING.md (hunk 1 above)
Comment (aditimaurya) [reply CONTRIBUTING.md:19]: hey, @thealphadollar! sorry, I just missed it. I'll fix it for sure. Thanks for your help!

**Comment by aditimaurya** [comment]
@thealphadollar, can you please check out the re-framed sentences? Please let me know if I need to work on them again.

File: README.md [hunk 3]
```diff
@@ -1,7 +1,6 @@
 IITKGP ERP AUTO LOGIN
 =====================
-
-A chrome extension that you have to setup just one time, to automatically login to your IITKGP ERP account every time. Just [install the chrome extension](https://chrome.google.com/webstore/detail/iitkgp-erp-auto-login/gdickphilmdekcbmpjmbnbikchaecbdk), and click on the extension logo (IIT Kharagpur's logo) to setup your login credentials.
+A chrome extension has to be set up just one time for automatic login to your IITKGP ERP account every time. Just [install the chrome extension](https://chrome.google.com/webstore/detail/iitkgp-erp-auto-login/gdickphilmdekcbmpjmbnbikchaecbdk), and click on the extension logo (IIT Kharagpur's logo) to setup your login credentials.
```
Comment (thealphadollar) [inline README.md:3]:
```suggestion
A chrome extension that has to be set up just one time for automatic login to your IITKGP ERP account every time. Just [install the chrome extension](https://chrome.google.com/webstore/detail/iitkgp-erp-auto-login/gdickphilmdekcbmpjmbnbikchaecbdk), and click on the extension logo (IIT Kharagpur's logo) to set up your login credentials.
```

**Review by thealphadollar — APPROVED** [review]
Yo @aditimaurya the changes look totally right except very very minor change :P I'm really glad you took my feedback and improved the PR considerably. Great going :rocket: :fire:

**Pull request merged**